src/data/faq.json

```
@@ -465,11 +465,21 @@
-                            "On the other hand, in case of previous basic immunization by two doses, the booster vaccination is declared as the 3rd of 3 vaccinations. By noting '3 of 3', the app correctly recognizes the booster vaccination as such and the 14-day waiting period for full vaccination protection is not displayed. Once a second booster vaccination has been administered with J&J after the basic immunization, the display of the fourteen day waiting period will be omitted as '3 of 3' will be entered as the number of the vaccination for this certificate.",
-                            "You can continue to show your basic immunization certificate as proof for these 14 days. This will retain its validity, provided that the technical or effective validity has not expired."
+                            "There are three cases where there may be a report of a fourteen-day waiting period after a booster vaccination:",
+                            "Case 1: Basic immunization with Johnson & Johnson (J&J) vaccine.",
+                            "Case 2: After recovery from COVID-19, a second booster vaccination is given.",
+                            "Case 3: After recovery from COVID-19, an initial vaccination with J&J is given.",
+                            "The reason is the limitation of the current EU guidelines for the creation of the different vaccination certificates/sequences, which is mapped accordingly by the apps. See: <a href='https://ec.europa.eu/health/sites/default/files/ehealth/docs/covid-certificate_json_specification_en.pdf'>https://ec.europa.eu/health/sites/default/files/ehealth/docs/covid-certificate_json_specification_en.pdf</a> (in English)",
+                            "Reason:",
+                            "Regarding case 1 and case 2:",
+                            "In the case of a basic immunization with vaccines other than J&J, such as BioNTech, Moderna, or AstraZeneca, the entry \"2 of 2\" for the second dose is made in the certificate under \"Number of vaccination\". However, regardless of the vaccine used, a waiting period of 14 days is always required after the last vaccination of the basic immunization, since full vaccine protection is only given after the formation of a sufficient amount of antibodies.",
+                            "As a result, checking apps, such as the CovPassCheck app, cannot distinguish whether the second vaccination procedure is a booster vaccination for the J&J basic immunization , or the second dose of the basic immunization. Therefore, the vaccination certificate for the booster vaccination will be displayed as invalid for 14 days.",

```

Remove extra space before comma.

```suggestion
                            "As a result, checking apps, such as the CovPassCheck app, cannot distinguish whether the second vaccination procedure is a booster vaccination for the J&J basic immunization, or the second dose of the basic immunization. Therefore, the vaccination certificate for the booster vaccination will be displayed as invalid for 14 days.",
```